Assets/Code/VehicleEditor/EditSystemDataAuthoring.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Linq;
+using Unity.Entities;
+using UnityEngine;
+
+/// <inheritdoc cref="EditSystemData"/>
+public class EditSystemDataAuthoring : MonoBehaviour {
+
+    [TextArea]
+    public string notes = "Did you make sure your new part \n- doesn't have nested children?\n- is 1x scale?";
```

Comment:
Theres a pretty neat free and MIT licensed package called Naughty attributes (https://github.com/dbrizov/NaughtyAttributes) that make things like this easier to make. Also has a bunch of other really cool editor attribute stuff. Not something that should be added with this PR btw, just a suggestion

Reply:
i agree but think that we should worry about niceties once we start on the full project